Sources/RedisMulti+SortedSet.swift

```diff
@@ -0,0 +1,387 @@
+/**
+ * Copyright IBM Corporation 2016
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Foundation
+
+/// Extend RedisMulti by adding the Sorted Set operations
+extension RedisMulti {
+    
+    /// Add an ZADD command to the "transaction"
+    ///
+    /// - Parameter key: The key.
+    /// - Parameter tuples: A list of tuples containing a score and value to be added to the sorted set.
+    @discardableResult
+    public func zadd(_ key: String, tuples: (Int, String)...) -> RedisMulti {
+        zaddArrayOfScoreMembers(key, tuples: tuples)
```

Comment:
Instead of calling the function `zaddArrayOfScoreMembers` and then `return self`, the code should be `return zaddArrayOfScoreMembers(...)`

This comment applies throughout the file for applicable APIs.